src/main/resources/templates/thymeleaf/review360.html

```diff
@@ -1,45 +1,45 @@
-<!DOCTYPE html >
-<html xmlns:th="http://www.thymeleaf.org">
-<head>
-    <title>Review 360</title>
-    <link href="/styles/adapter.css" rel="stylesheet">
-    <script type="application/javascript" src="/js/adapter.js"></script>
-</head>
-<body>
-    <div>
+    <!DOCTYPE html >
+    <html xmlns:th="http://www.thymeleaf.org">
+    <head>
+        <title>Review 360</title>
+        <link href="/styles/adapter.css" rel="stylesheet">
```

Comment:
```suggestion
        <link href="/styles/adapter.css" rel="stylesheet" />
```
Красоты для. Вообще можно весь html провалидировать.